common/app/common/commercial/CommercialProperties.scala

```diff
@@ -14,15 +14,12 @@ case class CommercialProperties(
   val isFoundationFunded: Boolean = branding(defaultEdition).exists(_.isFoundationFunded)
   def isSponsored(edition: Edition): Boolean = branding(edition).exists(_.isSponsored)
 
-  // Todo: remove this helper method when server-side ad targeting implemented before 2017-04-30
-  val surgeBuckets: String = adTargeting(defaultEdition).getOrElse("su", "")
-
   def branding(edition: Edition): Option[Branding] = for {
     editionBranding <- editionBrandings.find(_.edition == edition)
     branding <- editionBranding.branding
   } yield branding
 
-  def adTargeting(edition: Edition): Map[String, String] = {
+  def adTargeting(edition: Edition): Map[String, Either[String, Set[String]]] = {
```

Comment:
First, is it necessary to know if it is a single value or multiple ones? Can you just store it in a `Set` whatever you have one or many? I don't know much about ad targeting but it looks like the only place they are used server-side is to write [commercialMetaData](https://github.com/guardian/frontend/blob/c9eb80205bf47a8c7432a168f52fb6ade3849f1b/common/app/views/support/JavaScriptPage.scala#L46)

If it is necessary to be able to differentiate:
Seeing the signature of this function I have no idea what Either[String, Set[String]] represents, unless I know about the `stringify` function in [EditionAdTargeting.scala](https://github.com/guardian/frontend/blob/4253d3397595e267950eec908bb6fb9eddc1db25/common/app/common/commercial/EditionAdTargeting.scala).

It would be more self explanatory to declare an simple ADT like this:
```
sealed trait AdTargetingValue
case class AdTargetingSingleValue(value: String) extends AdTargetingValue
case class AdTargetingMultipleValues(values: Set[String]) extends AdTargetingValue
```

then you can carry around `Map[String, AdTargetingValue]` and a simple look at the definition of AdTargetingValue would tell any developer who is looking at the code in the future what it is all about.

Let me know if I'm missing something?